internal/tiltfile/tiltfile_state.go

```diff
@@ -591,10 +592,33 @@ func (s *tiltfileState) assembleK8sV1() error {
 
 }
 
-type claimPair struct {
-	resource       *k8sResource
-	selector       k8sObjectSelector
-	selectorString string
+func entitiesToFullName(entities []k8s.K8sEntity) (map[string]k8s.K8sEntity, error) {
```

Comment:
This name to me implies a `map[k8sEntity]FullName`. You might try `mapFullNameToEntity` or `entitiesByFullName`